js_assets/Question.vue

```diff
@@ -17,22 +17,19 @@
 
                     <button v-if="!hintGiven" :style="styleQuestion" class="btn btn-outline-secondary" @click="helpSteps">Hint</button>
                     <div v-if="hintGiven">
-                        <h3 class="text-muted">Hints</h3>
-                        <br>
-                            <h4 :style="styleMessage">{{ questionPackage.help_steps[0]['prompt'] }}</h4>
-                        <br>
-                            <h6 v-if="showAnswer">{{ questionPackage.help_steps[0]['answer'] }}</h6>
-                        <br>
-                            <button v-if="showAnswerButton" :style="styleQuestion" class="btn btn-outline-secondary" @click="showAnswer = true; showAnswerButton = false; showNextHintButton = true;">Show Hint's Answer</button>
-                        <br>
-                            <button v-if="hintLength > 1 && showNextHintButton" :style="styleQuestion" class="btn btn-outline-secondary" @click="showNextHint = true; showNextAnswerButton = true; showNextHintButton = false; hintCount++;">Show Next Hint</button>
-                        <br>
-                            <h4 v-if="showNextHint" :style="styleMessage">{{ questionPackage.help_steps[hintIndex]['prompt'] }}</h4>
-                        <br>
-                            <h6 v-if="showNextAnswer">{{ questionPackage.help_steps[hintIndex]['answer'] }}</h6>
-                        <br>
-                            <button v-if="showNextAnswerButton" :style="styleQuestion" class="btn btn-outline-secondary" @click="showNextAnswer = true; showNextAnswerButton = false;">Show Hint's Answer</button>
-                        <br>
+                    <h3 class="text-muted">Hints</h3>
+                        <div v-for="(hint, index) in hints">
+
+                            <br>
+                                <h4 :style="styleMessage">{{ hint['prompt'] }}</h4>
+                            <br>
+                                <h6 v-if="hintIndex > index">{{ hint['answer'] }}</h6>
+
+                                <button v-if="hintIndex === index" :style="styleQuestion" class="btn btn-outline-secondary" @click="hintIndex++; showNextHintButton = true;">Show Hint's Answer</button>
+                            <br>
+                                <button v-if="hintIndex === (index + 1) && showNextHintButton && hintLength > 1" :style="styleQuestion" class="btn btn-outline-secondary" @click="helpSteps">Show Next Hint</button>
+                            <br>
```

Comment:
If I click through the hints, I do not see a "next question" button.

Reply:
Confirming we're cool keeping the current behavior that user #1) has to attempt actually answering question to get next question button #2) they could at any time hit Random Question on navbar